### ISSUE TITLE
About the SNPs of functional importance

### ISSUE THREAD

**Issue by JielinLi** [Narasimhan-Lab/1000-genomes-natural-selection#9]
Hello, could you please provide more information on how the 4,9000 SNPs of functional importance were selected when calculating the P value? Thank you!

**Comment by mktle** [comment]
Hi, we used the SNPs that were designated as "potentially functional" sites in Mathieson et al. 2015: https://www.nature.com/articles/nature16152

They describe their process in the "Ancient DNA analysis" Methods subsection

**Comment by JielinLi** [comment]
Yes, I have read the paper, but the SNPs identified in it are different from the ones I encountered. For instance, the paper mentions selecting 9116 'Immunochip' SNPs as 'potentially functional' out of 1240K positions. However, in my analysis, I found 74711 common SNPs between the 'Immunochip' and 1240K datasets. Therefore, I would like to inquire if any additional screening processes were conducted subsequently?
